### ISSUE TITLE
Delay in page rendering

### ISSUE THREAD

**Issue by fbricon** [redhat-developer/openshift-dd-ext#66]
When opening the Openshift extension page, there's a noticeable delay before anything is displayed, 
![Jul-12-2022 11-45-47](https://user-images.githubusercontent.com/148698/178462187-222beab5-5929-4d33-a16e-454838b1b489.gif)

Could be because we're not leveraging recoil + suspense to load the context gracefully

**Comment by dgolovin** [comment]
@fbricon Probably it is because of all the loading we do under the hood for kube config and docker images. UI elements like current context and image selector should be shown right away, but in 'loading' state.

**Comment by fbricon** [comment]
Yes that's what I think. Loading through recoil would help I think

**Comment by itsCheithanya** [comment]
Can I work on this issue and could you be more specific about the implementation

**Comment by fbricon** [comment]
@itsCheithanya thanks for volunteering. I did start some experiments a while ago: https://github.com/fbricon/openshift-dd-ext/commit/aeeb78aca9b5219184258206f01456bb923ec448. Code might not compile/work and still have some outstanding issues, but that should give you a good starting point.

**Comment by itsCheithanya** [comment]
![Screenshot from 2022-10-06 20-58-58](https://user-images.githubusercontent.com/85927700/194354938-7f2df87e-5df9-45a8-a4e4-6c08ebdd6ebd.png)


 I built and installed the extension but when I  started it on local server its giving a blank page,How do I go further now?
